### ISSUE TITLE
npm install error "fatal error: json-c/json.h: No such file or directory"

### ISSUE THREAD

**Issue by deadprogram** [intel-iot-devkit/mraa#555]
I am unable to install the mraa 1.2.0 release under Ubuntu 14.04 using Node.js 4.2.2

The following error output is received:

```
../src/json/jsonplatform.c:31:25: fatal error: json-c/json.h: No such file or directory
 #include <json-c/json.h>
                         ^
compilation terminated.
make: *** [Release/obj.target/mraa/src/json/jsonplatform.o] Error 1
```

Thanks!

**Comment by Hbrinj** [comment]
Also are you possibly compiling the json platform loading version?

**Comment by Hbrinj** [comment]
Do you have json c installed? Are you by Chance trying to using imraa? How are you running cmake?

**Comment by arfoll** [comment]
Try 1.2.1 😋

**Comment by tingleby** [comment]
Node 6 seems good at 1.2.1 too!
```
tom@heim ~/test-npm % node -v
v6.3.1
tom@heim ~/test-npm % npm -v
3.10.5
```

**Comment by alext-mkrs** [comment]
This should be fixed and re-fixed :smiley: in 1.2.3, so closing this one.
